docs/tools/remote-signer-validation.md

```diff
@@ -0,0 +1,137 @@
+# tm-signer-harness
```

Comment:
is this a duplicate file with tools/tm-signer-harness/README.md ?

Reply:
Done 👍